src/Cache/Engine/FileEngine.php

```diff
@@ -413,8 +411,16 @@ protected function _setKey($key, $createKey = false)
      *
      * @return bool
      */
-    protected function _active()
+    protected function _dirWritable()
```

Comment:
@rchavik is totally right. Same goes about method signature at such low levels like ObjectRegistry.